### ISSUE TITLE
USWDS - Bug: Focus leaps from combo box to Chrome "Update" button menu

### ISSUE THREAD

**Issue by kerns-accessibility** [uswds/uswds#5291]
### Describe the bug

Focus leaps from combo box to the Chrome’s Update button menu when using standard JAWS screen-reader keyboard commands designed for combo boxes. 

Bug happens repeatedly, but not every single time. Doesn't occur in Firefox or Edge -- only occurs in Chrome.

### Steps to reproduce the bug

Utilizing JAWS screen-reader and Chrome, using Alt and up/down arrows ^  v  

a) Open combo box: Alt + v

b) Select list item: ^  v  

c) Close combo box: Alt + ^

d) Then open combo box again to select a different option.  

It often leaps to the Chrome Update button menu (but not always), and then gets stuck in the top webpage elements.

### Expected Behavior

After closing the combo box, the focus should remain on the combo box and be able to directly navigate to the content below the combo box.

### System setup

Operating System & Browser:
  - JAWS Version#: 2020.2012.13 ILM 
  - Google Chrome

### Additional context

Guidelines at issue

WCAG 2.1
•	2.4.3 Focus Order
•	3.2 Predictable: Make web content appear and operate in predicable ways.
•	3.2.3 Consistent navigation elements

Section 508
•	1194.21(c): Visual Focus 
•	1194.31(a): Use without vision 
•	302.9: Limited cognitive and learning abilities

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amycole501** [comment]
I'm unable to reproduce the behavior using the combobox example on this page https://designsystem.digital.gov/components/combo-box/ but it may be because this isn't 
 an issue that arises each time. 

However, on a related note, when I try to close the combo box using the "alt+up arrow key" combination, I hear JAWS say "close list box" but nothing happens (the box doesn't close in Firefox nor in Chrome).

**Comment by chandracarney** [comment]
In June, we discussed this needs testing triage with JAWS by an accessibility tester. Early thoughts:
- If it's valid, could we trap focus? 
- If it's a Chrome browser bug, we'd want to to open an issue with Chrome.

Resolving this issue will require working with an accessibility testing subject matter expert who can test with necessary physical devices and physical tools.
